app/scripts/modules/core/src/delivery/create/CreatePipelineButton.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import * as ReactGA from 'react-ga';
+import { BindAll } from 'lodash-decorators';
+
+import { Application } from 'core/application';
+import { ReactInjector } from 'core/reactShims';
+import { Tooltip } from 'core/presentation/Tooltip';
+
+import { CreatePipelineModal } from './CreatePipelineModal';
+
+export interface ICreatePipelineButtonProps {
+  application: Application;
+}
+
+export interface ICreatePipelineButtonState {
+  showCreatePipelineModal: boolean;
+}
+
+@BindAll()
+export class CreatePipelineButton extends React.Component<ICreatePipelineButtonProps, ICreatePipelineButtonState> {
+  constructor(props: ICreatePipelineButtonProps) {
+    super(props);
+
+    this.state = {
+      showCreatePipelineModal: false,
+    };
+  }
+
+  private showCallBack(showCreatePipelineModal: boolean) {
+    this.setState({showCreatePipelineModal});
+  }
+
+  private createPipeline() {
+    ReactGA.event({category: 'Pipelines', action: `Create Pipeline`});
```

Comment:
why backticks?

Reply:
The place I copy pasted it from had a variable and I didn't change to quotes.